### PR TITLE
update latest samples to 1.0.0-rc1-update1

### LINKED ISSUE #1155: samples/latest/HelloWeb inconsistent Dockerfile
### Latest samples contains inconsistent Dockerfile

`samples/latests/HelloWeb` contains inconsistent `Dockerfile`
latest folder contains 1.0.0-rc1 instead of 1.0.0-rc1-update1
### Functional impact

`samples/latest/HelloWeb` docker container fail to start
### Minimal repro steps

```
cd samples/latest/HelloWeb
docker build -t aspnet-home-helloweb .
docker run -it -p 80:5004 aspnet-home-helloweb
```
### Expected result

Container should start kester server
### Actual result

Container started but dnx fail to run `kestrel` command.
`Error: Unable to load application or execute command 'kestrel'. Available commands: web.`
### Further technical details

Looks like `samples/latest` should be switched to `1.0.0-rc1-update1` or deleted.

### PR DESCRIPTION
**Pull request by sfilatov**
`samples/latest` updated to 1.0.0-rc1-update1.
This commit fixes #1155

### CODE REVIEW

**Comment by Eilon** [comment]
@glennc ?

**Comment by linquize** [comment]
ping

**Comment by Kagamine** [comment]
I think the latest means the aspnetvnext feed in myget, not the last stable in nuget.

File: samples/latest/NuGet.Config [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetvnext/api/v2" />
     <add key="NuGetorg" value="https://nuget.org/api/v2/" />
+    <add key="AspNetRC1" value="https://www.myget.org/F/aspnetrc1/api/v2/" />
```
Comment (moozzyk) [inline samples/latest/NuGet.Config]: You should not need these - these packages are already on NuGet

File: samples/latest/HelloWeb/project.json [hunk 2]
```diff
@@ -10,15 +10,15 @@
         "**.vspscc"
     ],
     "dependencies": {
-        "Microsoft.AspNet.Server.Kestrel": "1.0.0-*",
-        "Microsoft.AspNet.IISPlatformHandler": "1.0.0-*",
-        "Microsoft.AspNet.Diagnostics": "1.0.0-*",
-        "Microsoft.AspNet.Hosting": "1.0.0-*",
-        "Microsoft.AspNet.StaticFiles": "1.0.0-*",
-        "Microsoft.Extensions.Logging.Console": "1.0.0-*"
+        "Microsoft.AspNet.Server.Kestrel": "1.0.0-rc1-final",
```
Comment (moozzyk) [inline samples/latest/HelloWeb/project.json]: `*` should have the same effect given the feeds NuGet.Config

**Comment by sfilatov** [comment]
Outdated. I will close because ongoing work #1277 will switch latest sample to current latest version

**Pull request closed without merging**